main.go

```diff
@@ -36,73 +31,74 @@ import (
 	"github.com/sapcc/hermes/internal/util"
 )
 
-var configPath *string
-
 func main() {
-	logg.ShowDebug = osext.GetenvBool("HERMES_DEBUG")
-	parseCmdlineFlags()
-
 	setDefaultConfig()
-	readConfig(configPath)
+	bindEnvVariables()
+
+	logg.ShowDebug = viper.GetBool("hermes.debug")
+
 	keystoneDriver := configuredKeystoneDriver()
 	storageDriver := configuredStorageDriver()
 	readPolicy()
 	must.Succeed(api.Server(keystoneDriver, storageDriver))
 }
 
-func parseCmdlineFlags() {
-	// Get config file location
-	configPath = flag.String("f", "hermes.conf", "specifies the location of the TOML-format configuration file")
-	flag.Usage = func() {
-		fmt.Fprintf(os.Stderr, "Usage of %s:\n", os.Args[0])
-		flag.PrintDefaults()
-	}
-	flag.Parse()
-}
-
 func setDefaultConfig() {
 	var nullEnforcer, err = policy.NewEnforcer(make(map[string]string))
 	if err != nil {
 		panic(err)
 	}
+
+	viper.SetDefault("hermes.debug", false)
+
 	viper.SetDefault("hermes.keystone_driver", "keystone")
 	viper.SetDefault("hermes.storage_driver", "elasticsearch")
 	viper.SetDefault("hermes.PolicyEnforcer", &nullEnforcer)
+	viper.SetDefault("hermes.PolicyFilePath", "etc/policy.json")
```

Comment:
```suggestion
	viper.SetDefault("hermes.PolicyFilePath", "/etc/policy.json")
```
I would recommend to use an absolute path here, using relative paths can be confusing